include/triton/Dialect/Triton/Transforms/ArithTypeConversion.h

```diff
@@ -0,0 +1,18 @@
+#ifndef TRITON_DIALECT_TRITON_TRANSFORMS_ARITH_TYPE_CONVERSION_H_
+#define TRITON_DIALECT_TRITON_TRANSFORMS_ARITH_TYPE_CONVERSION_H_
+#include "mlir/Transforms/DialectConversion.h"
+
+namespace mlir::arith {
```

Comment:
Probably shouldn't put this in arith's namespace.
```suggestion
namespace mlir::triton {
```

Reply:
My rough plan is to upstream this into LLVM. But you're right it's probably best to put in the triton namespace for now.